src/main/java/com/gmail/nossr50/config/skills/salvage/SalvageConfig.java

```diff
@@ -27,6 +28,11 @@ public SalvageConfig(String fileName) {
     protected void loadKeys() {
         salvageables = new ArrayList<Salvageable>();
 
+        if (!config.isConfigurationSection("Salvageables")) {
+            mcMMO.p.getLogger().severe("Could not find Salvageables section in " + fileName);
+            return;
+        }
+
```

Comment:
Technically for a micro-optimization you could move this above the creation of the empty array list object